src/main/scala/is/hail/sparkextras/RepartitionedOrderedRDD2.scala

```diff
@@ -13,21 +13,30 @@ import org.apache.spark.sql.Row
   * needed. No assumption should need to be made about partition keys, but currently
   * assumes old partition key type is a prefix of the new partition key type.
   */
+object RepartitionedOrderedRDD2 {
```

Comment:
Can we rename this RepartitionOrderedRVD please?

Reply:
The naming is my fault. I was trying to follow what existing conventions I could see. Are you more objecting to the "ed" or the "RDD2", or both? The "ed" I don't have strong feelings on, I just thought it should be a noun. But it isn't an RVD. Or are you just referring to the companion object?

Reply:
Just the RDD2.  RDD2 was what we called the thing that became RVD.  I thought I went through and renamed all the classes, but I guess I missed some.

Reply:
I thought you were talking about just the object, which takes the RVD and returns an RDD.

I'd like to avoid renaming the RDD2 for now, because it actually is an RDD and not an RVD because it relies on the Spark dependency/partition infrastructure.

Reply:
Oh, I've been thinking RVD means subclass of `RVD`, which has an `RDD[RegionValue]` field, but that an `RDD[RegionValue]` is not an RVD.